todos/src/main/java/com/maksimpegov/todos/models/ProcessResponse.java

```diff
@@ -0,0 +1,38 @@
+package com.maksimpegov.todos.models;
+
+import java.util.Collections;
+import java.util.List;
+
+public class ProcessResponse {
```

Comment:
What means 'ProcessResponse'? Mybe better use 'TodoServiceResponse'? Just curious... Oh you have TodoService responce already.... So why then TodoService returning ProcessResponse? And controller returning TodoServiceResponse? )))

Reply:
TodoServiceResponse is form that whole micro-service return, ProcessResponse - is response that service gives to controller after all data work and processed